core/src/com/fallenflame/game/LightController.java

```diff
@@ -128,15 +131,15 @@ public void initialize(PlayerModel player, ExitModel exit,
         rayhandler.setAmbientLight(0, 0, 0, AMBIENT_LIGHT);
         rayhandler.setBlur(true);
         rayhandler.setBlurNum(3);
-
         updateCamera();
 
         // Save player and config.
         this.player = player;
         this.lightingConfig = levelLighting;
-
+        playerLightOffset = (player.getLightRadius()/2) * 0.4f;
+        flareLightOffset = 0.5f;
```

Comment:
Why are constants being defined in initialize()? You should just make both of these vars constants defined above.

Reply:
The flare one can be a constant, but the player one cannot because it needs to change based on the current value of the light radius.

Reply:
I meant more the flareLightOffset and the multiplier for playerLightOffset, but it's not the biggest deal if you just leave the playerLightOffset as is